cms/static/js/views/active_video_upload_list.js

```diff
@@ -212,13 +231,16 @@ define([
             },
 
             showErrorMessage: function(errorMsg) {
-                var titleMsg = gettext('Your file could not be uploaded');
-                this.fileErrorMsg = new NotificationView.Error({
-                    title: titleMsg,
-                    message: errorMsg
-                });
-                this.fileErrorMsg.show();
-                this.readMessages([titleMsg, errorMsg]);
+                var titleMsg;
+                if (!this.fileErrorMsg) {
+                    titleMsg = gettext('Your file could not be uploaded');
+                    this.fileErrorMsg = new NotificationView.Error({
+                        title: titleMsg,
+                        message: errorMsg
+                    });
+                    this.fileErrorMsg.show();
+                    this.readMessages([titleMsg, errorMsg]);
```

Comment:
@mushtaqak Are you sure the following two line will go inside `IF`? If yes, it means this message will only work if ` if (!this.fileErrorMsg)` condition is `true`
```
this.fileErrorMsg.show();
this.readMessages([titleMsg, errorMsg]);
```